source/_includes/puppet-collections/_rpm_command_output.md

```diff
@@ -0,0 +1,5 @@
+    puppetlabs-release-pc1-fedora-22.noarch.rpm:
+        Header V4 RSA/SHA1 Signature, key ID ef8d349f: OK
```

Comment:
I've manually updated the key ID in the output; I did not generate new output. If anything else changes in this output, I need either updated output or instructions on how to generate updated output.